### DISCUSSION TITLE
Week 10 Technical Demo Loops

### DISCUSSION THREAD

**Discussion by eculler** [earthlab-education/Earth-Analytics-2023-01-Intro#562] [Technical Demo Videos]
[Technical Demo video](https://cuboulder.zoom.us/rec/share/aIdlYFpJXewI0QUfOxadloBGSrkHWF2jH1cwqQyVqbZF6jva-Rb52SggIEHbOm9j.W1xiS6JvQU_z1QZP)

**Comment by taylor-obrien** [comment]
Is this assignment due Tuesday 11/7?

**Comment by Adriannaluger** [comment]
@eculler Do you have a new eta for the video? Thanks!

**Comment by eculler** [comment]
@earthlab-education/ea-students @earthlab-education/ea-instructors 

Sorry everyone! I thought I posted the video last week. It's up now - my bad! The assignment will be due Friday instead.

Also! I had a ton of trouble with Codespaces on this one. If you are also having trouble, please let me know. We may need to take a pause as a class and get you all access to an environment on your computers.

**Reply by edchanonline** [reply]
My kernal keeps crashing during `merge_arrays`. Curious if that's the trouble you are also having! Thank you!

**Reply by annettebalmes** [reply]
You have to upgrade your machine type to merge. Elsa was able to change it to 4-core and that worked. I went straight to 8-core and that worked for me.

**Reply by edchanonline** [reply]
Thanks @annettebalmes !! I got it working yesterday!!
